tox.ini

```diff
@@ -47,6 +47,7 @@ deps =
     ecos >=2.0.8
     highspy >=1.1.2.dev3
     jaxopt >=0.8.3
+    kvxopt >=1.3.2
```

Comment:
We should do the same for `[testenv]` as well to test other platforms like macOS or Python (coverage is only run on `ubuntu-latest`).

When some platforms don't work they can be disabled, for instance `py{39,310}-{linux,macos}: piqp` above only tests PIQP on Linux and macOS with Python 3.9 and 3.10.

Reply:
Most likely omitted :sweat_smile: